src/test/java/seedu/pennywise/logic/parser/AddCommandParserTest.java

```diff
@@ -121,12 +123,27 @@ public void parse_allFieldsPresent_success() {
 
     @Test
     public void parse_optionalFieldsMissing_success() {
-        // zero tag
+        // zero tags
```

Comment:
Might it not be zero tags? It appears to have `TAG_MOVIE`

Reply:
oh whoops, let me change that

Reply:
i think this test case was left over from when we could attach multiple tags to an entry, ill remove it entirely since it doesn't fit our app now.